modules/nw-multus-creating-whereabouts-reconciler-daemon-set.adoc

```diff
@@ -29,14 +29,18 @@ Use the following procedure to deploy the `whereabouts-reconciler` daemon set.
 $ oc edit network.operator.openshift.io cluster
 ----
 
-. Modify the `additionalNetworks` parameter in the CR to add the `whereabouts-shim` network attachment definition. For example:
+. Include the `additionalNetworks` section shown in this example YAML extract within the `spec` definition of the custom resource (CR):
 +
 [source,yaml]
 ----
 apiVersion: operator.openshift.io/v1
 kind: Network
 metadata:
+  creationTimestamp: "2024-01-03T11:19:12Z"
```

Comment:
Personally, I wouldn't include this information in the example YAML file here. It's inviting potential user error with copy and paste in my opinion.

Is this information supposed to be written by hand or generated by a tool? I see references to a generation number, resource version, and UID, all of which are unlikely to be the same between what the user is doing and what the YAML example here displays.

Reply:
Thanks @kowen-rh I am going to look at that again and maybe just detail the exact snippet of YAML that needs included